microsoft-edge/webview2/reference/javascript/chrome.yml

```diff
@@ -0,0 +1,23 @@
+### YamlMime:TSType
+name: Chrome
+uid: 'WebView2Script!Chrome:class'
+package: WebView2Script!
+fullName: Chrome
+summary: ''

```

Comment:
Need to include a summary, which is also displayed on the top level page in the JS ref docs. 

https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/javascript/?branch=pr-en-us-2315